docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -217,6 +217,13 @@ A schema collection consists of a MongoDB document for a table.
         }
     }
 
+
+.. note::
```

Comment:
I feel like we shouild keep it as a seperate note for the user as it's more of a note or point of interest for the user rather than an explicit description of the field. What do you guys think ?

Reply:
I think you are overusing note in this doc already. Just have it as a normal paragraph.